content/reports/1000_IBM Power - Virtual Machines/011_All PowerHMC VMs.yaml

```diff
@@ -0,0 +1,86 @@
+---
+  title: IBM PowerHMC VMs
+  rpt_group: Custom
+  rpt_type: Custom
+  priority:
+  db: ManageIQ::Providers::InfraManager::Vm
+  cols:
+  - name
+  - os_image_name
+  - ram_size
+  - entitled_processors
+  - processor_pin_policy
+  - num_cpu
+  - num_disks
```

Comment:
My test environments aren't showing any disk info in inventory (and I don't see any errors when I run refresh, which include some disk info [here](https://github.com/ManageIQ/manageiq-providers-ibm_power_hmc/blob/master/app/models/manageiq/providers/ibm_power_hmc/inventory/parser/infra_manager.rb#L45-L57)).

Are you able to verify `num_disks` and `disk_size` work using your HMC?

Reply:
Hi @jaywcarman, on my side I can populate disks column because it is linked to disks table. Most of the case we have 0; but for example if one VM has a Logical Unit it can display it

For example here : 
![image](https://user-images.githubusercontent.com/70648532/164450884-3e392a46-b2b9-4c32-ab08-1f4f83a4e00e.png)

But `disk_size` stills unpopulated. Maybe wrong information inside my table for `size` column, or wrong format : 
```
vmdb_development=# select device_name, size from disks where id=1;
 device_name |    size
-------------+-------------
 callistelu0 | 10737418240
```

However I have the information on the interface : 
![image](https://user-images.githubusercontent.com/70648532/164451910-3cc87230-c3f4-4455-9e20-57f4ce405b77.png)

Reply:
For PowerVS we're using `hardware.num_disks` and `hardware.disks.size` - might be worth trying them here.
https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/content/reports/1000_IBM%20Power%20-%20Virtual%20Machines/010_All%20Power%20VMs.yaml#L31-L32